ReactiveObjCBridgeTests/ObjectiveCBridgingSpec.swift

```diff
@@ -300,5 +316,20 @@ class ObjectiveCBridgingSpec: QuickSpec {
 				}
 			}
 		}
+		
+		describe("RACSubscriber.sendNext") {
```

Comment:
Totally, very important to guarantee this invariant against changing Swift versions.